app/controllers/api/application_controller.rb

```diff
@@ -9,4 +9,16 @@ def set_headers
   def verify_json_request
     not_found unless request.format.json?
   end
+
+  def render_server_error(exception)
+    render json: { error: "Internal server error", message: exception.message }, status: :internal_server_error
+  end
+
+  def render_not_found
+    render json: { error: "The given ID does not match any vacancy for your ATS" }, status: :not_found
+  end
+
+  def render_bad_request
+    render json: { error: "Request body could not be read properly" }, status: :bad_request
+  end
```

Comment:
If these responses are only used by (and bind to) the ATS API, could we move the method definitions to the ATS API Vacancies controller?